app/components/build-message.js

```diff
@@ -9,129 +9,130 @@ const { escapeExpression: escape } = Ember.Handlebars.Utils;
 export default Component.extend({
   readableMessage: computed('message.code', 'message.args', function () {
```

Comment:
Since we are now depending on `message.key`, we should probably add it here as well, like this:
```readableMessage: computed('message.code', 'message.key', 'message.args', function () {```

This way if message.key changes for some reason, the computed property will know to recompute on next access. Here's more info about Ember Computed Properties if it's helpful :smile: 
https://guides.emberjs.com/release/object-model/computed-properties/

Reply:
done in https://github.com/travis-ci/travis-web/commit/537c39e16409e256068999a3fe8361547cb31d7f

Reply:
Awesome, it looks good :) thanks!